native/cocos/scene/Camera.cpp

```diff
@@ -110,7 +111,6 @@ void Camera::destroy() {
         _window = nullptr;
     }
     _name.clear();
-    _geometryRenderer->destroy();
```

Comment:
Why not destroy it in destroy function?

Reply:
The destroy function of Camera is never called.

Reply:
If so, then the logic is error. We should fix the logic. Currently, destructor invoke destroy function is better.

Reply:
1

Reply:
1